README.md

```diff
@@ -1,137 +1,3 @@
-blobStore.createContainerInLocation(null, "mycontainer");
-
-// add blob
-blob = blobStore.blobBuilder("test").payload("testdata").build();
-blobStore.putBlob("mycontainer", blob);
-```
-
-BlobStore Example (Clojure):
-
-```clojure
-(use 'org.jclouds.blobstore2)
-(def *blobstore* (blobstore "azureblob" account encodedkey))
-(create-container *blobstore* "mycontainer")
-(put-blob *blobstore* "mycontainer" (blob "test" :payload "testdata"))
-```
-
-ComputeService
----------------
-Simplifies the task of managing machines in the cloud. For example, you can use ComputeService to start 5 machines and install your software on them.
-
-Compute Example (Java):
-
-```java
-// init
-context = new ComputeServiceContextFactory().createContext(
-"aws-ec2",
-accesskeyid,
-secretaccesskey,
-ImmutableSet.of(new Log4JLoggingModule(), new SshjSshClientModule()));
-
-client = context.getComputeService();
-
-// define the requirements of your node
-template = client.templateBuilder().osFamily(UBUNTU).smallest().build();
-
-// setup a boot user which is the same as your login
-template.getOptions().runScript(AdminAccess.standard());
-
-// these nodes will be accessible via ssh when the call returns
-nodes = client.createNodesInGroup("mycluster", 2, template);
-
-// you can now run ad-hoc commands on the nodes based on predicates
-responses = client.runScriptOnNodesMatching(inGroup("mycluster"), "uptime", wrapInInitScript(false));
-```
-
-Compute Example (Clojure):
-
-```clojure
-(use 'org.jclouds.compute2)
-
-; create a compute service using sshj and log4j extensions
-(def compute (*compute* "trmk`-ecloud" "user" "password" :sshj :log4j))
-
-; launch a couple nodes with the default operating system, installing your user.
-(create-nodes *compute* "mycluster" 2
-(TemplateOptions$Builder/runScript (AdminAccess/standard)))
-
-; run a command on that group 
-(run-script-on-nodes-matching *compute* (in-group? "mycluster") "uptime" 
-(RunScriptOptions$Builder/wrapInInitScript false))
-```
-
-Check out https://github.com/jclouds/jclouds-examples for more examples!
-
-Downloads
-------------------------
-* release notes: http://www.jclouds.org/documentation/releasenotes/1.5
-* installation guide: http://www.jclouds.org/documentation/userguide/installation-guide
-* maven repo: http://repo2.maven.org/maven2 (maven central - the default repository)
-* snapshot repo: https://oss.sonatype.org/content/repositories/snapshots
- 
-Resources
-----------------------------
-* Project page: http://jclouds.org/
-* Documentation: http://www.jclouds.org/documentation/index
-* Community: http://www.jclouds.org/documentation/reference/apps-that-use-jclouds 
-* User group: http://groups.google.com/group/jclouds
-* Dev group: http://groups.google.com/group/jclouds-dev
-* Twitter: http://twitter.com/jclouds
-
-
-License
--------
-Copyright (C) 2009-2012 jclouds, Inc.
-
-Licensed under the Apache License, Version 2.0
+jclouds moved to Apache in 2013 and archived this repository.  Please visit our
```

Comment:
[minor] Add link(s) to the mailing list and/or new project site?

Reply:
Done.